six/modules/c++/six.sicd/tests/test_read_sicd_from_vts.cpp

```diff
@@ -0,0 +1,321 @@
+/* =========================================================================
```

Comment:
So I’m not really sure how to set this up. When I had the test one of the generated files,  it looks like VTS tries to do an extra layer of validation and fails. Is there someplace I could put some real SICDs for Jenkins to pull from for running this?

From: Adam Sylvester [mailto:notifications@github.com]
Sent: Friday, July 22, 2016 6:10 PM
To: ngageoint/six-library <six-library@noreply.github.com>
Cc: Jonathan Means <Jonathan.Means@mdaus.com>; Author <author@noreply.github.com>
Subject: Re: [ngageoint/six-library] Fix how CSM figures out an images type, to match how NITFReadControl now does it. (#90)


In six/modules/c++/six.sicd/tests/test_read_sicd_from_vts.cpp<https://github.com/ngageoint/six-library/pull/90#discussion_r71953317>:

> @@ -0,0 +1,321 @@

> +/* =========================================================================

Can you add something to the automated tests to call this?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/ngageoint/six-library/pull/90/files/b5fa7dce12abfbd03f84ce03aee18d4669cf64de#r71953317>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AFRAYPNif0UN_8d6g6SASiO2fmtatDimks5qYT_NgaJpZM4JNnoK>.


The information contained in this communication is confidential, is intended only for the use of the recipient(s) named above, and may be legally privileged. If the reader of this message is not the intended recipient, you are hereby notified that any dissemination, distribution, or copying of this communication is strictly prohibited. If you have received this communication in error, please re-send this communication to the sender and delete the original message or any copy of it from your computer system.